var/spack/repos/builtin/packages/openfst/package.py

```diff
@@ -0,0 +1,68 @@
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Openfst(AutotoolsPackage):
+    """OpenFst is a library for constructing, combining, optimizing, 
+        and searching weighted finite-state transducers (FSTs). Weighted 
+        finite-state transducers are automata where each transition has 
+        an input label, an output label, and a weight."""
+
+    homepage = "http://www.example.com"
+    url      = "http://www.openfst.org/twiki/pub/FST/FstDownload/openfst-1.6.1.tar.gz"
+    list_url = "http://www.openfst.org/twiki/bin/view/FST/FstDownload"
+
+    version('1.6.1',  '1173066ed987072183b950b54bdc9315')
+    version('1.6.0',  '2b7bcfde6b629857dc5f9ad7edd6ece9')
+    version('1.5.4',  'e52bd59ec9d9edde0c1268a382662741')
+    version('1.5.3',  'f04e580b5bc574571854304c9656a3e2')
+    version('1.5.2',  'e9d43874f7cadf791394caab3925eee4')
+    version('1.5.1',  '8869e44c5a4af65409ae78b9f482b40e')
+    version('1.5.0',  'a24fee5ffe28744c6fb7b1a49e0006c4')
+    version('1.4.1',  'ca8f1730b9b9b281e515611fa9ae23c0')
+    version('1.4.0',  '662367ec91084ffab48ee9b5716de39c')
+    version('1.3.4',  '6517be1b7e794a578440f344e50b58bc')
+    version('1.3.3',  'c7ba9e791eba501bb9d5b95ccc6e5231')
+    version('1.3.2',  '9b0d777f177d9917bb93adef19b7098a')
+    version('1.3.1',  '9d1ce908eac610da805dfd4fdc337b82')
+    version('1.2.11', 'bc755b8481cd8946e85115c016dbb168')
+    version('1.2.10', '2c73dca4cbfe3850b9b7f6988249c870')
+    version('1.2.9',  'e8ce0d74b05c175c12f3637002b19132')
+    version('1.2.8',  '822e74b2ab5693b02a3267b558658e2d')
+    version('1.2.7',  '97196a97d2a1ec88d612321e64dac2e4')
+    version('1.2.6',  'cbb2b7421d4ba36b329242041057b98f')
+    version('1.2.5',  'a2b5559f8d1e4e880a571558e8983b27')
+    version('1.2.4',  '849996eba8afd162014716b48a5c2667')
+    version('1.2.3',  'a245f9aedbe851b180b2160f90819bce')
+    version('1.2.2',  'f5cc59e7678a8d0c9ce4d0647f9f4c8d')
+    version('1.2.1',  '6d55480771034afcd87dac998e737b84')
+    version('1.2',    'efee97019b45d26c0e804045688a2692')
+    version('1.1',    '7491c12e0878ab594cc14ae88103c486')
+    version('1.0',    'da8120d5d8dd3e66c9779419f9f4fdee')
```

Comment:
You don't have to add all of these versions if you don't want to. I generally only add the latest version because that's all I need and care to support.